build.sbt

```diff
@@ -16,7 +16,9 @@ scalacOptions := Seq(
 libraryDependencies ++= Seq(
   "com.azavea"                  %% "scaliper"              % "0.5.0-SNAPSHOT" % "test",
   "io.dylemma"                  %% "xml-spac"              % "0.3-SNAPSHOT",  // Published locally
-  "org.apache.spark"            %% "spark-core"            % "2.1.0",  // TODO add "provided" later
+  "org.apache.spark"            %% "spark-core"            % "2.2.0",  // TODO add "provided" later
```

Comment:
Well, it should solve the fact that when you do `sbt assembly`, to run it against some spark submit (e.g. EMR), that it doesn't bake in the Spark and Hadoop dependencies, because those are provided by `spark-submit`.

I'm testing on the project I needed that for now, and it looks like the assembly is actually still baking in spark and hadoop; so actually I'll need to find a better solution.

The thing that this is trying to solve is, when these dependencies are marked "provided", `sbt run` will error with ClassNotFound's. This attempts to put the correct files on the class path in the "run" situation, so that doesn't happen.